components/src/components/Common/Simple.edit.data.ts

```diff
@@ -28,9 +28,9 @@ export default [
         valueProperty: 'value',
         data: {
             custom(context) {
-                var values = [];
+                const values = [];
                 values.push({ label: 'Any Change', value: 'data' });
-                context.utils.eachComponent(context.instance.options.editForm.components, function(component, path) {
+                context.utils.eachComponent(context.instance.options.editForm.components, (component, path) => {
                     if (component.key !== context.data.key) {
                         values.push({
                             label: component.label || component.key,
```

Comment:
Is this related to the date time control? Not sure what this change is doing

Reply:
I ran the Linter and this was just a Linting issue that needed to be resolved.